bittide/src/Clash/Cores/Xilinx/SystemMonitor.hs

```diff
@@ -0,0 +1,266 @@
+-- SPDX-FileCopyrightText: 2024 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE NoFieldSelectors #-}
+
+module Clash.Cores.Xilinx.SystemMonitor (
+  Status (..),
+  temperatureMonitorCelcius,
+  temperatureMonitor,
+  adcToTemperature,
+) where
+
+import Clash.Prelude
+
+import Clash.Cores.Xilinx.Xpm.Cdc.Internal
+import Clash.Functor.Extra ((<<$>>))
+
+{- | A wrapper for a SYSMONE1 instance which only monitors the FPGA temperature. Converts the ADC
```

Comment:
Please change `ADC value` into `analog measurement` or something alike.
ADC stands for Analog-Digital Converter.

This goes for all other mentions of ADC as analog measurement